scripts/gobuster.sh

```diff
@@ -5,5 +5,9 @@ git clone https://github.com/oj/gobuster /opt/gobuster
 
 cd /opt/gobuster && \
   make && \
-  cp /opt/gobuster/gobuster /usr/local/bin/gobuster
-
+  cp /opt/gobuster/gobuster /usr/local/bin/gobuster && \
+  git clone --no-checkout https://github.com/v0re/dirb.git && \
+  cd dirb && \
+  git config core.sparseCheckout true && \
+  echo  wordlists/ > .git/info/sparse-checkout && \
+  git checkout master
```

Comment:
Move the wordlists directory into `/opt/gobuster` and then clean up dirb.

```suggestion
  git checkout master && \
  cd /opt && \
  mv dirb/wordlists gobuster/wordlists && \
  rm -rf dirb
```